.github/workflows/new-event-notify.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     timeout-minutes: 2
     steps:
       - name: Send Telegram notification for new issue
-        uses: sehat1137/telegram-notifier@27e8505551e7806d06c7e97e41e86ac7d548b04b # v1.4.0
+        uses: sehat1137/telegram-notifier@e99082adf06006d67e381310f0447fae4eb8be31 # v1.4.0
```

Comment:
please, next time - don't delete existing releases. create new ones. this should have been `1.4.1`